core-plugins/widgets/FTP-batchsource.json

```diff
@@ -30,6 +30,18 @@
           "widget-type": "json-editor",
           "label": "File System Properties",
           "name": "fileSystemProperties"
+        },
+        {
+          "widget-type": "select",
+          "label": "Ignore Non-Existing Folders",
+          "name": "ignoreNonExistingFolders",
+          "widget-attributes": {
+            "values": [
+              "false",
```

Comment:
While we have the values listed both ways in our plugins, let's keep the ordering as true, false instead of false, true. I'll file a jira to fix the other ones. https://issues.cask.co/browse/HYDRATOR-1144